src/main/resources/liquibase/tenant/scripts/2020-09-03--12-00-add-case-insensitive-unique-constraints.xml

```diff
@@ -0,0 +1,29 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+
+  <changeSet  id="2020-08-28--12-30-drop-liquibase-unique-constraint-from-both-set_spec-and-name-columns" author="Illia Daliek">
+    <dropUniqueConstraint tableName="set_lb"
+                          schemaName="${database.defaultSchemaName}"
+                          constraintName="set_spec_unique_constraint"
+                          uniqueColumns="set_spec"/>
+    <dropUniqueConstraint tableName="set_lb"
+                          schemaName="${database.defaultSchemaName}"
+                          constraintName="name_unique_constraint"
+                          uniqueColumns="name"/>
+  </changeSet>
+
+  <changeSet  id="2020-08-28--11-00-add-case-insensitive-unique-constraint-to-both-set_spec-and-name-columns" author="Illia Daliek">
+    <sql>
+      CREATE UNIQUE INDEX set_spec_unique_constraint
+      ON ${database.defaultSchemaName}.set_lb (lower(set_spec));
+    </sql>
+    <sql>
+      CREATE UNIQUE INDEX name_unique_constraint
+      ON ${database.defaultSchemaName}.set_lb (lower(name));
```

Comment:
is `lower(...)` good enough here?  What about accents or other diacritical marks?

e.g. `facade` and `Facade` would not be unique, but `façade` would be.

Reply:
It looks like you are right. There is, however,  a critical need for this functionality to be merged for the upcoming demo. I suggest that we merge it now and we'll create a ticket to investigate this point further.

Reply:
Quote from PO:
Our PO doesn’t insist on handling such cases and even says that there are a lot of words in French, German etc, where the same word with and without (or other) diacritical mark, umlaut and so on can have another meaning (e.g. tâche ‘task’ — tache ‘spot’), so it is arguable, whether it should be considered unique or not, unlike case sensitivity.

Reply:
It's worth noting that this deviates from the vast majority of the project's storage modules.  I wasn't part of the decision making process and can't speak to the rationale behind the decision.